sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/key_curve_name.hpp

```diff
@@ -28,15 +30,28 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys {
      *
      * @param value The string value of the instance.
      */
-    KeyCurveName(std::string const& value)
+    explicit KeyCurveName(std::string value)
```

Comment:
I checked the other ext-enums we have exposed to the public and we are following this pattern:
- Take the input arg as a copy and move it to the field.

I changed it to follow this pattern.. (not sure if we have guideline for this), but it makes sense to do it like this because we are automatically documenting that we are making a copy of the input string.  On the other approach:
- Take a std::string const& and then copy it to the field
The caller would wonder if the passed string const& needs to be alive and will be use for all the life of the ext-enum.